gateway/src/apicast/policy/token_introspection/apicast-policy.json

```diff
@@ -13,13 +13,10 @@
         "description": "Introspection Endpoint URL",
```

Comment:
When using the OIDC endpoint, this setting is also unnecessary, right?

Reply:
But this endpoint is discoverable through OIDC.

For example: https://keycloak.3sca.net/auth/realms/oidc-demo/.well-known/openid-configuration.

It should be possible to get it from `context.proxy.oauth.config`. You can use `require('resty.repl').start()` to start a REPL and see around the objects available.